modin/experimental/engines/omnisci_on_ray/frame/calcite_algebra.py

```diff
@@ -11,44 +11,130 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
+"""
+Module provides classes for relational algebra expressions.
+
+Provided classes set reflects relational algebra format used by
+OmniSci engine.
+"""
+
 import abc
 from .expr import BaseExpr
 
 
 class CalciteInputRefExpr(BaseExpr):
-    """Caclcite version of input column reference.
+    """
+    Caclcite version of input column reference.
```

Comment:
Do you mean I should mention this class is derived from `BaseExpr`? Shouldn't documentation generate this info basing on class declaration?

Reply:
Yes, something about classes interconnections. Not sure `sphinx` can do this, @YarShev do you know a way we can add 
automatic notion about base class during building docs from docstrings?

Reply:
I am not aware of that but we can do as it is [here](https://github.com/modin-project/modin/blob/master/modin/engines/python/pandas_on_python/frame/partition_manager.py#L23), i.e. put a line like """This class implements the interface in `<base_class>`.""". Then, after generation of the rst doc from docstring we can see respective description.

Reply:
@ienkovich i think it will be problematic to add automatic notion about class interconnections by `sphinx`, it is better to add something like [this](https://github.com/modin-project/modin/blob/master/modin/engines/python/pandas_on_python/frame/partition_manager.py#L23) manually.

Reply:
There is the `:show-inheritance:` directive which should make `sphinx` include proper inheritance info into generated documentation.

BTW the line you referenced seems to have a bug ('method' instead of 'class').

Reply:
I think it's appropriate solution and in that way there is no need in additional notions about base classes in the docstrings themselves.